Component/Pet/Pet.js

```diff
@@ -2,30 +2,67 @@ import styled from "styled-components";
 
 const StyledPet = styled.div`
   display: flex;
-  border: 2px solid #000000;
+  border: 5px solid #000000;
   border-radius: 10px;
-  margin: 10px;
-  padding: 10px;
-  justify-content: space-around;
+  margin: 2% 0%;
+  padding: 10px 3%;
   align-items: center;
-  max-width: 250;
   background-color: var(--secondary-color);
+  width: 50vw;
+  height: 20vh;
+  min-height: 150px;
+  gap: 3%;
+
+  @media screen and (max-width: 1000px) {
+    width: 70vw;
+  }
+
   span {
-    font-size: 20vw;
+    font-size: 12vh;
     position: absolute;
+    left: 20px;
+
+    @media screen and (max-width: 1000px) {
+      font-size: 10vh;
+    }
   }
   .portrait {
```

Comment:
As mentioned in my previous review, I suggest to avoid nested styles. Additionally, you shouldn't mix styled components with CSS classes. Please use only styled components for this project.